docs/user_manual/orders.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: "11. Orders"
+section: 11
+---
+
+Once a pool is ready for sequencing, there maybe delay to batch multiple pools
+for a single flow cell and a pool maybe placed in multiple lanes. To facilitate
+this, MISO has the idea of an order that indicates how much sequencing needs to
+be performed and whether the required sequencing has been completed.
+
+Orders are based on a pool, sequencing platform, and the sequencing parameters.
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=section sub=sub section-title=section-title title="Active Orders List" %}
+
+The active orders page shows all the orders and the number of matching
+sequencing runs. From the "Tracking" list, click "Active Orders". There are
```

Comment:
But the active orders page doesn't show all orders... what about something like `The active orders page shows all the orders for which sequencing is either outstanding or in progress.` for the first sentence?